.gitignore

```diff
@@ -60,5 +60,5 @@ crashlytics-build.properties
 
 # CUSTOM EXCLUDES
 .vscode/
-*node_modules
-out/
\ No newline at end of file
+__pycache__/
```

Comment:
I see a `__pycache__` commited, might have been commited before this gitignore was added, but either way I'd remove it from git and make sure this rule is working as you expect

Reply:
Yeah I realized it was being commited and then added it to the gitignore. Just ran `git ls-tree -r HEAD` and `__pycache__` is present, does that mean I'm good?

Reply:
honestly not sure.  I typically rely on my IDE to grey-out gitignored files for me.